### PR TITLE
x11-themes/*: cleaning

### PR DESCRIPTION
**Pull request by epsilon-0**
also taking proxy maintainership cuz why not

### CODE REVIEW

File: x11-themes/chameleon-xcursors/chameleon-xcursors-0.5-r1.ebuild [hunk 1]
```diff
@@ -1,7 +1,8 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
+inherit xdg-utils
```
Comment (telans) [inline x11-themes/chameleon-xcursors/chameleon-xcursors-0.5-r1.ebuild:5]: Why not use `xdg.eclass` and remove the `pkg_` phases?

Comment (epsilon-0) [reply x11-themes/chameleon-xcursors/chameleon-xcursors-0.5-r1.ebuild:5]: :open_mouth: 
I was not aware of this one, thanks a lot!
Gonna update them.

File: x11-themes/chameleon-xcursors/chameleon-xcursors-0.5-r1.ebuild [hunk 2]
```diff
@@ -1,7 +1,8 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
+inherit xdg
```
Comment (a17r) [inline x11-themes/chameleon-xcursors/chameleon-xcursors-0.5-r1.ebuild:5]: empty line after EAPI pls

File: x11-themes/echo-icon-theme/echo-icon-theme-0.3.89.0_pre20081031.ebuild [hunk 3]
```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2013 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit gnome2-utils
+EAPI=7
+inherit xdg
```
Comment (a17r) [inline x11-themes/echo-icon-theme/echo-icon-theme-0.3.89.0_pre20081031.ebuild:5]: empty line after EAPI pls

File: x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild [hunk 4]
```diff
@@ -1,26 +1,22 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-inherit gnome2-utils
+EAPI=7
+inherit xdg
```
Comment (a17r) [inline x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild:5]: empty line after EAPI pls

File: x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild [hunk 5]
```diff
@@ -1,26 +1,22 @@
...
 LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="amd64 ~arm ~arm64 x86"
-IUSE="minimal"
 
-RDEPEND="!minimal? ( x11-themes/adwaita-icon-theme )
-	x11-themes/hicolor-icon-theme"
+RDEPEND="x11-themes/hicolor-icon-theme"
 
 S=${WORKDIR}/${PN}-${PV%.*}
```
Comment (a17r) [inline x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild:21]: You're adding quotes over WORKDIR elsewhere, let's be consistent.

File: x11-themes/gargantuan-icon-theme/gargantuan-icon-theme-1.7.ebuild [hunk 6]
```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-inherit xdg-utils
+EAPI=7
+inherit xdg
```
Comment (a17r) [inline x11-themes/gargantuan-icon-theme/gargantuan-icon-theme-1.7.ebuild:5]: empty line after EAPI pls

File: x11-themes/nou-icon-theme/nou-icon-theme-09.02.08.ebuild [hunk 7]
```diff
@@ -1,24 +1,18 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-inherit versionator xdg-utils
+EAPI=7
+inherit xdg
```
Comment (a17r) [inline x11-themes/nou-icon-theme/nou-icon-theme-09.02.08.ebuild:5]: empty line after EAPI pls

File: x11-themes/nou-icon-theme/nou-icon-theme-09.02.08.ebuild [hunk 8]
```diff
@@ -1,24 +1,18 @@
...
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="minimal"
-
-RDEPEND="!minimal? ( || ( x11-themes/tango-icon-theme x11-themes/adwaita-icon-theme ) )"
-DEPEND=""
-
-RESTRICT="binchecks strip"
 
 S=${WORKDIR}
```
Comment (a17r) [inline x11-themes/nou-icon-theme/nou-icon-theme-09.02.08.ebuild:17]: You're adding quotes over WORKDIR elsewhere, let's be consistent.

File: x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-18.12.01.ebuild [hunk 9]
```diff
@@ -13,15 +13,10 @@ LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="amd64 x86"
 
-DEPEND="x11-themes/numix-icon-theme"
-RDEPEND="${DEPEND}"
+RDEPEND="x11-themes/numix-icon-theme"
 
 src_install() {
 	insinto /usr/share/icons
 	doins -r Numix-Circle{,-Light}
```
Comment (a17r) [inline x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-18.12.01.ebuild:21]: useless empty line

File: x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.01.24.ebuild [hunk 10]
```diff
@@ -13,15 +13,10 @@ LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="amd64 x86"
 
-DEPEND="x11-themes/numix-icon-theme"
-RDEPEND="${DEPEND}"
+RDEPEND="x11-themes/numix-icon-theme"
 
 src_install() {
 	insinto /usr/share/icons
 	doins -r Numix-Circle{,-Light}
```
Comment (a17r) [inline x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.01.24.ebuild:21]: useless empty line

File: x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.02.07.ebuild [hunk 11]
```diff
@@ -13,15 +13,10 @@ LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="amd64 x86"
 
-DEPEND="x11-themes/numix-icon-theme"
-RDEPEND="${DEPEND}"
+RDEPEND="x11-themes/numix-icon-theme"
 
 src_install() {
 	insinto /usr/share/icons
 	doins -r Numix-Circle{,-Light}
```
Comment (a17r) [inline x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.02.07.ebuild:21]: useless empty line

File: x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.02.22.ebuild [hunk 12]
```diff
@@ -13,15 +13,10 @@ LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="amd64 x86"
 
-DEPEND="x11-themes/numix-icon-theme"
-RDEPEND="${DEPEND}"
+RDEPEND="x11-themes/numix-icon-theme"
 
 src_install() {
 	insinto /usr/share/icons
 	doins -r Numix-Circle{,-Light}
```
Comment (a17r) [inline x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.02.22.ebuild:21]: useless empty line

File: x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.03.20.ebuild [hunk 13]
```diff
@@ -13,15 +13,10 @@ LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="amd64 x86"
 
-DEPEND="x11-themes/numix-icon-theme"
-RDEPEND="${DEPEND}"
+RDEPEND="x11-themes/numix-icon-theme"
 
 src_install() {
 	insinto /usr/share/icons
 	doins -r Numix-Circle{,-Light}
```
Comment (a17r) [inline x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.03.20.ebuild:21]: useless empty line

File: x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.04.15.ebuild [hunk 14]
```diff
@@ -13,15 +13,10 @@ LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="amd64 x86"
 
-DEPEND="x11-themes/numix-icon-theme"
-RDEPEND="${DEPEND}"
+RDEPEND="x11-themes/numix-icon-theme"
 
 src_install() {
 	insinto /usr/share/icons
 	doins -r Numix-Circle{,-Light}
```
Comment (a17r) [inline x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.04.15.ebuild:21]: useless empty line

File: x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.05.07.ebuild [hunk 15]
```diff
@@ -13,15 +13,10 @@ LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="amd64 x86"
 
-DEPEND="x11-themes/numix-icon-theme"
-RDEPEND="${DEPEND}"
+RDEPEND="x11-themes/numix-icon-theme"
 
 src_install() {
 	insinto /usr/share/icons
 	doins -r Numix-Circle{,-Light}
```
Comment (a17r) [inline x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.05.07.ebuild:21]: useless empty line

File: x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.09.20.ebuild [hunk 16]
```diff
@@ -13,15 +13,10 @@ LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="amd64 x86"
 
-DEPEND="x11-themes/numix-icon-theme"
-RDEPEND="${DEPEND}"
+RDEPEND="x11-themes/numix-icon-theme"
 
 src_install() {
 	insinto /usr/share/icons
 	doins -r Numix-Circle{,-Light}
```
Comment (a17r) [inline x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.09.20.ebuild:21]: useless empty line

File: x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.12.27.ebuild [hunk 17]
```diff
@@ -13,15 +13,10 @@ LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
-DEPEND="x11-themes/numix-icon-theme"
-RDEPEND="${DEPEND}"
+RDEPEND="x11-themes/numix-icon-theme"
 
 src_install() {
 	insinto /usr/share/icons
 	doins -r Numix-Circle{,-Light}
```
Comment (a17r) [inline x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.12.27.ebuild:21]: useless empty line

File: x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-20.06.07.ebuild [hunk 18]
```diff
@@ -13,15 +13,10 @@ LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
-DEPEND="x11-themes/numix-icon-theme"
-RDEPEND="${DEPEND}"
+RDEPEND="x11-themes/numix-icon-theme"
 
 src_install() {
 	insinto /usr/share/icons
 	doins -r Numix-Circle{,-Light}
```
Comment (a17r) [inline x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-20.06.07.ebuild:21]: useless empty line

File: x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-20.07.11.ebuild [hunk 19]
```diff
@@ -13,15 +13,10 @@ LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
-DEPEND="x11-themes/numix-icon-theme"
-RDEPEND="${DEPEND}"
+RDEPEND="x11-themes/numix-icon-theme"
 
 src_install() {
 	insinto /usr/share/icons
 	doins -r Numix-Circle{,-Light}
```
Comment (a17r) [inline x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-20.07.11.ebuild:21]: useless empty line

File: x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-20.09.19.ebuild [hunk 20]
```diff
@@ -0,0 +1,22 @@
...
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="x11-themes/numix-icon-theme"
+
+src_install() {
+	insinto /usr/share/icons
+	doins -r Numix-Circle{,-Light}
+
```
Comment (a17r) [inline x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-20.09.19.ebuild:21]: useless empty line

File: x11-themes/nuovo-icon-theme/nuovo-icon-theme-0.5-r1.ebuild [hunk 21]
```diff
@@ -1,22 +1,16 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-inherit xdg-utils
+EAPI=7
+inherit xdg
```
Comment (a17r) [inline x11-themes/nuovo-icon-theme/nuovo-icon-theme-0.5-r1.ebuild:5]: empty line after EAPI pls

File: x11-themes/nuovo-icon-theme/nuovo-icon-theme-0.5-r1.ebuild [hunk 22]
```diff
@@ -1,22 +1,16 @@
...
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="minimal"
-
-RDEPEND="!minimal? ( x11-themes/adwaita-icon-theme )"
-DEPEND=""
-
-RESTRICT="binchecks strip"
 
 S=${WORKDIR}
```
Comment (a17r) [inline x11-themes/nuovo-icon-theme/nuovo-icon-theme-0.5-r1.ebuild:15]: You're adding quotes over WORKDIR elsewhere, let's be consistent.

File: x11-themes/yasis-icon-theme/yasis-icon-theme-0.4.2-r1.ebuild [hunk 23]
```diff
@@ -1,22 +1,16 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-inherit xdg-utils
+EAPI=7
+inherit xdg
```
Comment (a17r) [inline x11-themes/yasis-icon-theme/yasis-icon-theme-0.4.2-r1.ebuild:5]: empty line after EAPI pls

**Review by a17r — CHANGES_REQUESTED** [review]
```$ eshowkw numix-icon-theme-circle
Keywords for x11-themes/numix-icon-theme-circle:
         |                             |   u   |  
         | a   a     p s     a   r     |   n   |  
         | m   r h   p p   s l i i m m | e u s | r
         | d a m p p c a x 3 p a s 6 i | a s l | e
         | 6 r 6 p p 6 r 8 9 h 6 c 8 p | p e o | p
         | 4 m 4 a c 4 c 6 0 a 4 v k s | i d t | o
---------+-----------------------------+-------+-------
18.08.29 | + o o o o o o + o o o o o o | 6 # 0 | gentoo
18.09.19 | + o o o o o o + o o o o o o | 6 #   | gentoo
18.10.03 | + o o o o o o + o o o o o o | 6 #   | gentoo
18.12.01 | + o o o o o o + o o o o o o | 6 #   | gentoo
19.01.24 | + o o o o o o + o o o o o o | 6 #   | gentoo
19.02.07 | + o o o o o o + o o o o o o | 6 #   | gentoo
19.02.22 | + o o o o o o + o o o o o o | 6 #   | gentoo
19.03.20 | + o o o o o o + o o o o o o | 6 #   | gentoo
19.04.15 | + o o o o o o + o o o o o o | 6 #   | gentoo
19.05.07 | + o o o o o o + o o o o o o | 6 #   | gentoo
19.09.20 | + o o o o o o + o o o o o o | 6 o   | gentoo
19.12.27 | ~ o o o o o o ~ o o o o o o | 7 #   | gentoo
20.06.07 | ~ o o o o o o ~ o o o o o o | 7 #   | gentoo
20.07.11 | ~ o o o o o o ~ o o o o o o | 7 o   | gentoo
```

Rather than making changes to all those ebuilds, isn't there a more substantial cleanup overdue here?

PS: The school of kde proj ebuild maintenance told me to also get rid of superflouous `A`/`An` filler words at the beginning of `DESCRIPTION`, of which there are plenty here.

**Comment by jonasstein** [comment]
I am dissolving desktop-misc, so any dev may merge this when it is ready. I did not reassign the package to m-n yet, so you do not have to update your PR.

File: x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild [hunk 24]
```diff
@@ -34,6 +31,7 @@ src_prepare() {
 		cp "${x}"/places/scalable/start-here-gentoo.svg \
 			"${x}"/places/scalable/start-here.svg || die
 	done
+	default
```
Comment (a17r) [inline x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild:34]: We inherit it, so let's run `xdg_src_prepare` here.

File: x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.09.20.ebuild [hunk 25]
```diff
@@ -1,27 +1,22 @@
...
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-inherit gnome2-utils
+inherit xdg
 
-DESCRIPTION="An icon theme for Linux from the Numix project"
+DESCRIPTION="icon theme from the Numix project"
```
Comment (a17r) [inline x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.09.20.ebuild:8]: (this and multiple other occurrences) Let's start `DESCRIPTION` with capital letters in general.

**Review by a17r — CHANGES_REQUESTED** [review]

File: x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.09.20.ebuild (hunk 25 above)
Comment (epsilon-0) [reply x11-themes/numix-icon-theme-circle/numix-icon-theme-circle-19.09.20.ebuild:8]: done

File: x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild (hunk 24 above)
Comment (epsilon-0) [reply x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild:35]: done

File: x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild [hunk 26]
```diff
@@ -1,30 +1,28 @@
...
-EAPI=5
-inherit gnome2-utils
+EAPI=7
+
+inherit xdg
 
 MY_P=${PN}_${PV}
 
 DESCRIPTION="A scalable icon theme called Faenza"
 HOMEPAGE="http://tiheum.deviantart.com/art/Faenza-Icons-173323228"
```
Comment (vaukai) [inline x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild:11]: https for HOMEPAGE 
( https://www.deviantart.com/tiheum/art/Faenza-Icons-173323228 )

**Review by Polynomial-C — APPROVED** [review]

File: x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild (hunk 26 above)
Comment (epsilon-0) [reply x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild]: done

File: x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild [hunk 27]
```diff
@@ -34,6 +32,7 @@ src_prepare() {
 		cp "${x}"/places/scalable/start-here-gentoo.svg \
 			"${x}"/places/scalable/start-here.svg || die
 	done
+	default
```
Comment (a17r) [inline x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild:35]: `xdg_src_prepare` is already calling `default`, don't do it twice.

**Review by a17r — CHANGES_REQUESTED** [review]

File: x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild (hunk 27 above)
Comment (epsilon-0) [reply x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild:35]: done

**Review by a17r — COMMENTED** [review]
Thank you very much!

**Pull request closed without merging**